app/src/main/java/com/example/trempelapp/presentation_layer/mainScreen/home_page_fragments/HomePageFragment.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.trempelapp.presentation_layer.mainScreen.home_page_fragments
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.example.trempelapp.R
+
+class HomePageFragment : Fragment() {
```

Comment:
Extend Base Fragment

Reply:
And inject dagger and Add architecture layers here for the future

Reply:
done